app/src/pages/Group/containers/CompetitionsTable.jsx

```diff
@@ -74,6 +74,14 @@ function sortCompetitions(competitions) {
   if (!competitions) return [];
 
   return competitions.sort((a, b) => {
+    if (a.status === b.status) {
```

Comment:
This is the same condition being checked on line 86, so if the code passes that comparison it means that the status are guaranteed to be the same because subtracting the indices will result in a falsy 0 value.

I think it would be better to not perform the status comparison twice.
Did you try to fit these changes into the below return block?

Here's something I came up with if you're looking for inspiration:
![image](https://user-images.githubusercontent.com/51417989/222914655-e39ea133-9487-49c5-b4d3-9daed6e8e21f.png)